### ISSUE TITLE
Health scaling. 1.12.2 modpack: RLCraft. Sponge causes issues.

### ISSUE THREAD

**Issue by TheParsley** [SpongePowered/SpongeForge#3016]
**I am currently running**

- SpongeForge version: RC 3844 Specific Sponge ore requires this version.
- SpongeVanilla version: 
- Java version: 8
- Operating System: 

- Plugins (13):Minecraft, Minecraft Coder Pack, SpongeAPI, Sponge, Block Monitor, CatClearLag, EpicBanItem, FESpongeCompat, Lockette, PermissionsEx, PlayerDataReset-Plugin, RedProtect, WorldEdit
- Mods (109): Minecraft, Minecraft Coder Pack, Forge Mod Loader, Minecraft Forge, SpongeAPI, Sponge, Advanced Fishing, Advanced Hook Launchers, AlcatrazCore, Antique Atlas, Anvil Patch - Lawful, Aquaculture, Armor Underwear, AutoRegLib, BNBGamingLib, Battle Towers, Baubles, Better Combat Rebirth, Better Survival, Bloodmoon, Bountiful Baubles, Campfire Spit, Carrots Lib, Carry On, Comforts, CompatSkills, Coralreef, Corpse Complex, CraftTweaker JEI Support, CraftTweaker2, Craftable Chainmail, Craftable Horse Armour [CHA&S], CreativeCore, CreativeCoreDummy, Defiled Lands, Disenchanter, Doomlike Dungeons, Dynamic Surroundings, Dynamic Trees, Dynamic Trees for Traverse, DynamicSurroundingsCore, Enhanced Visuals, Familiar Fauna, First Aid, FoamFix, FoamFixCore, Food Expansion, Forge Endertech, Forge Essentials, Forgotten Items, Global GameRules, Grappling hook mod, Ice and Fire, Infernal Mobs, Inventory Tweaks, It's the little things, ItemPhysic, IvToolkit, Just Enough Items, LLibrary, Level Up! Reloaded, Lost Souls, Lycanites Mobs, Mo' Bends, No Tree Punching, OreLib Support Mod, Phosphor Lighting Engine, Potion Core, Potion Fingers, Quality Tools, Realistic Torches, Reborn Core, Recurrent Complex, Reskillable, Rin's So Many Enchantments?, Roguelike Dungeons, Rough Tweaks, Ruins Mod, Scaling Health, Serene Seasons, Shadowfacts' Forgelin, Silent Lib, Simple Camp Fire, Sit, Spartan Fire, Spartan Shields, Spartan Weaponry, SpawnerControl, SpongeForge, SwingThroughGrass, Switch-Bow, TNTUtils, TNTUtils Core, The Lost Cities, TickProfiler, Tool Belt, Tough As Nails, Traverse, Trinkets and Baubles, Trumpet Skeleton, TschippLib, Tumbleweed, Varied Commodities, Waystones, Wearable Backpacks, Wolf Armor and Storage, XP From Harvest, XP Tome, bacodifficulty

**Issue Description**
Health scaling. 1.12.2 modpack: RLCraft. Heart containers are adding 2 hearts instead of one. Causing the second one to go through a damaging cycle.

**Comment by LIGITPICKLE** [comment]
Was there any fixe for this problem? I only seem to have this happen to me when I try to add Sponge onto my server. No other plugins have been added but once Sponge gets added this problem occurs.

**Comment by TheParsley** [comment]
The issue was never fixed and as far as i know it hasn't been fixed. Feel free to open a new issue as this is considered necro.

**Comment by dualspiral** [comment]
No, do not create a new issue. If the issue was considered “necro”, we would have closed it. The comment was helpful, as it at least reminded us it was here, we do miss issues on occasion.

**Comment by gabizou** [comment]
So, to add complexity, this isn't because Sponge is causing issues, it's that there's never been a clean way to handle extra containers from mods. It'll be a while before I can create any sort of changes or investigate what changes will be needed to support whatever mod is adding health containers (it'll help if you find out which mod is doing it, because at this moment, all I know is that it's one of the dozens of mods in this mudpack, and I'm not going to spend the time to figure out which mod it is, there's other things that I know what I need to do more than what this issue is giving me).

Not to say it won't get solved, but it's low priority as my time is limited. The more information you can provide, the faster it'll get done (if I knew exactly what modifications they were making, then I can write cases to handle those modifications).

**Comment by LIGITPICKLE** [comment]
The mod that adds the Heart Containers is names Scaling Health. If it might be needed the mod that makes your individual body parts have separate health values is called First Aid. Reply if this is at all helpful or you need something else.
